server/db-patches/patch-22-23.sql

```diff
@@ -0,0 +1,14 @@
+ALTER TYPE shot_block_type ADD VALUE IF NOT EXISTS 'watchdog';
+
+CREATE TABLE watchdog_submissions (
+  id SERIAL,
+  shot_id CHARACTER VARYING(270) NOT NULL,
+  request_id CHARACTER(36) NOT NULL,
+  nonce CHARACTER(36) NOT NULL,
+  positive_result boolean,
+  CONSTRAINT watchdog_pkey PRIMARY KEY (id),
+  CONSTRAINT watchdog_shot_id_fkey FOREIGN KEY (shot_id)
+      REFERENCES data(id)
+      ON UPDATE NO ACTION
+      ON DELETE CASCADE
```

Comment:
@chenba I'm not sure we should ever delete items from this table, even if the parent shot is deleted. Will the upstream watchdog service preserve everything that would be legally needed in such a case?

Reply:
The plan is to not delete the shot until it's been reviewed manually.  As for preserving data for legal reasons, I'll let @clouserw answer.

Reply:
A watchdog client shouldn't need to hold on to anything if it has confidence in the answer it received.

Since we're just starting this, we want to review the images in case of false positives.  In that case, we should flag and remove from public access, but not delete until we hear back from legal.  In the future, we might just auto-delete, so, let's not make it too complex.  Making it an SQL query we have to run to delete flagged images would be fine.